internal/contributors/contributors.py

```diff
@@ -0,0 +1,55 @@
+import pandas as pd
+import toml
+
+## Config
+with open("internal/contributors/contributors.toml") as f:
```

Comment:
Why do we need to factor out the config file? Maybe just hard-code the values in this script, for simplicity?
(One less thing to change.)

Reply:
Right, deleting the `.toml` file.